src/entryPoints/hyperswitch/AcceptInvite.res

```diff
@@ -0,0 +1,139 @@
+@react.component
+let make = () => {
+  open HSwitchUtils
+  open APIUtils
+  open LogicUtils
+
+  let updateDetails = useUpdateMethod()
+  let merchantDataFromLocalStorage =
+    LocalStorage.getItem("accept_invite_data")
```

Comment:
Can we use `getInfoFromLocalStorage` func to fetch the value of accept_invite_data

Reply:
Not needed it's only required in this file